src/js/facility-locator/components/markers/TownHall.jsx

```diff
@@ -0,0 +1,30 @@
+import DivMarker from './DivMarker';
+import React, { Component, PropTypes } from 'react';
+
+class TownHall extends Component {
+  render() {
+    const style = {
+      backgroundColor: '#2e8540',
+      borderRadius: '50%',
+      marginTop: '-12px',
+      marginLeft: '-12px',
+      width: '24px',
+      height: '24px',
+      zIndex: '999',
+    };
```

Comment:
I think it's probably enough to just make up some class name that won't conflict with anything else, but I do see the appeal of trying to isolate the styles from the rest of the page…